packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -213,9 +213,11 @@ export class InsuredBridgeL1Client {
     return this.bridgePools[l1Token];
   }
 
-  getBridgePoolForL2Token(l2Token: string): BridgePoolData {
-    const bridgePoolData = Object.values(this.bridgePools).filter((bridgePool) => bridgePool.l2Token == l2Token)[0];
-    if (!bridgePoolData) throw new Error(`No bridge pool initialized for ${l2Token}`);
+  getBridgePoolForL2Token(l2Token: string, chainId: string): BridgePoolData {
+    const bridgePoolData = Object.values(this.bridgePools).filter(
```

Comment:
nit: find will only return one value, making this a little simpler

Reply:
+1 fixed